src/config/selectors.ts

```diff
@@ -426,3 +403,11 @@ export const IMAGE_THUMBNAIL_UPLOADER = 'imageThumbnailUploader';
 export const REMOVE_THUMBNAIL_BUTTON = 'removeThumbnailButton';
 
 export const MUI_CHIP_REMOVE_BTN = 'CancelIcon';
+export const HOME_LOAD_MORE_BUTTON_SELECTOR = '[role="feed"]';
+export const buildItemsGridMoreButtonSelector = (id: string): string =>
+  `#${buildItemCard(id)} [data-testid="MoreVertIcon"]`;
+export const buildItemMenuId = (id: string): string => `item-menu-id-${id}`;
+export const SORTING_SELECT_SELECTOR_TEST_ID = 'sortingSelect';
+export const SORTING_SELECT_SELECTOR = `[data-testid="${SORTING_SELECT_SELECTOR_TEST_ID}"]`;
+export const SORTING_ORDERING_SELECTOR_DESC = '.lucide-arrow-up-wide-narrow';
+export const SORTING_ORDERING_SELECTOR_ASC = '.lucide-arrow-down-narrow-wide';
```

Comment:
It is possible to use data-cy here ? what if lucide decide to change their classes ?